src/services/VerifyAccountRequestProcessor.ts

```diff
@@ -151,7 +156,7 @@ export class VerifyAccountRequestProcessor {
   		this.personIdentityTableName,
   	);
 
-  	const copCheckResult = this.calculateCopCheckResult(verifyResponse);
+  	const copCheckResult = await this.calculateCopCheckResult(verifyResponse);
```

Comment:
Hmm `calculateCopCheckResult` isn't async is it?

Reply:
Rebase mistake, fixed